05.wc/wc.rb

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  options = ARGV.getopts('lwc')
+  options.transform_values! { true } if options.values.all? { |value| value == false }
```

Comment:
```suggestion
  options.transform_values! { true } if options.values.all? { |value| !value }
```

`== false`と書くのは冗長なのでこんな風に書くこともできます